common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -27,15 +29,46 @@ def setUp(self):
         """
         super(AccountSettingsPageTest, self).setUp()
 
-        AutoAuthPage(self.browser, username=self.USERNAME, password=self.PASSWORD, email=self.EMAIL).visit()
+        self.user_id = AutoAuthPage(
+            self.browser, username=self.USERNAME, password=self.PASSWORD, email=self.EMAIL
+        ).visit().get_user_id()
 
+    def visit_account_settings_page(self):
```

Comment:
The only test that doesn't use it is ``test_link_on_dashboard_works``. I was thinking more about this and I think it would be better if I introduced a new test subclass for all the basic settings page tests, and then I could call the method in the setup.

With the learner profile changes, I found that I preferred having the explicit call to set things up which returned the new page. It can get difficult to manage these tests when there are too many class variables and it isn't clear what has been set up and hasn't. In this case I'm okay with making the change.